tests/specs/send/send-stx.spec.ts

```diff
@@ -25,6 +26,32 @@ test.describe('send stx: tests on testnet', () => {
     await sendPage.selectStxAndGoToSendForm();
   });
 
+  test('that we show high fee warning in case of high custom fee', async ({ sendPage, page }) => {
+    const highFeeAmount = HIGH_FEE_AMOUNT_STX + 1;
+    await sendPage.amountInput.fill(amount);
+    await sendPage.amountInput.blur();
+    await sendPage.recipientInput.fill(TEST_TESTNET_ACCOUNT_2_STX_ADDRESS);
+    await sendPage.recipientInput.blur();
+
+    await page.getByTestId(SharedComponentsSelectors.MiddleFeeEstimateItem).click();
+    await page.getByTestId(SharedComponentsSelectors.CustomFeeSelectItem).click();
+    await page
+      .getByTestId(SharedComponentsSelectors.CustomFeeFieldInput)
+      .fill(highFeeAmount.toString());
+
+    await sendPage.previewSendTxButton.click();
+
+    await page.waitForTimeout(4000);
+    await page.getByTestId(SendCryptoAssetSelectors.HighFeeWarningDialog).isVisible();
+    await page.waitForTimeout(4000);
+
+    await page.getByTestId(SendCryptoAssetSelectors.HighFeeWarningDialogSubmit).click();
+    await page.waitForTimeout(4000);
```

Comment:
Ideally it'd be great if we avoid use of timeout like this. Doesn't the auto-wait feature of Playwright ensure that it doesn't fail immediately?

Reply:
I think sometimes the auto-wait itself can cause the tests to hang. We should investigate removing the timeouts to optimise the tests.

Maybe we can do it in https://github.com/leather-wallet/extension/issues/3919 and re-visit them all seperately?

Reply:
good catch, thanks 
I just used it for local work, and forgot to remove 😕